cachito/web/models.py

```diff
@@ -416,7 +416,7 @@ def _validate_request_package_configs(request_kwargs, pkg_managers_names):
         )
 
     # Validate the values for each package manager configuration (e.g. packages.npm)
-    valid_package_config_keys = {"path"}
+    valid_package_config_keys = {"path", "requirements_build_files", "requirements_files"}
```

Comment:
Should `"requirements_build_files", "requirements_files"` only be allowed for pip?